src/main/resources/stl/mutation_report.st

```diff
@@ -7,7 +7,7 @@
 \</head>
 
 \<body onload="loadMutationCountBar()">
-  \<img src="mutant-swarm-logo.png"/>
+  \<img src="logo.png"/>
```

Comment:
How hard would it be to do this the other way around? i.e. generate the png called "mutant-swarm-logo.png"? I don't really mind either way, just curious if it was obvious how to do that. @JayGreeeen any thought on preferable approach?

Reply:
You can just change the name of the file in resources/img, no?

Reply:
I can change the name in **ReportGenerator** and the file name located in **resources** if you prefer.

![image](https://user-images.githubusercontent.com/1533999/76231196-41644f80-61fb-11ea-8263-81b1c77a974b.png)

![image](https://user-images.githubusercontent.com/1533999/76231298-6bb60d00-61fb-11ea-92f1-bd9725bae8b2.png)

Reply:
@JayGreeeen any preference on which way to do it? If not then let's just go with this?

Reply:
no, preference. Good to have it as logo.png 👍🏻